installer/conf/omsagent.d/baseline.conf

```diff
@@ -0,0 +1,30 @@
+# Baseline plugins
+
+<source> 
+    type exec
+    tag oms.baseline_immediate_run
+    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d 

```

Comment:
you need to include the omsbaseline into installer/datafiles/base_omsagent.data. otherwise, it will not exist in the bundle.